assets/translations/Russian

```diff
@@ -109,15 +109,15 @@ Language=Язык
 Last launched program's output=Сообщения последнего запущенного ярлыка
 Launching=Запуск
 Link CPU clock=Частота для ярлыков
-Link created=
+Link created=Ярлык создан
 Link description=Описание ярлыка
 Link label=Метки ярлыков
 Link title=Заголовок ярлыка
 List Body=Фон списка
 Loading.. Please wait..=Идёт загрузка.. Ждите..
 Loading.=Загрузка.
 Loading=Загрузка
-Log Viewer=Просмотрщик журнала
```

Comment:
I've intentionally change it to singular, why revert to plural?

Reply:
If the "журнал" will be in the plural form, "журналов", it will be in the general style of naming other 'viewers'. They are now all in the plural: "Просмотрщик изображений/текстов/журналов". Same for 'installers' - all in the plural: "Установщик пакетов/тем оформления". In other words - uniformity.
And in my sight and hearing it looks and sounds better (more familiar, more expected) in plural form.
By analogy, even if using an image viewer we will constantly view only the one same single image, we will still call it "Просмотрщик изображений", in the plural form.
In the singular, it would be more logical to shorten the name to simply "Журнал" (i.e., drop the beginning word "Просмотрщик "). I have already proposed this option earlier. Short and concise, and no loss of meaning in this particular case.
So it's not fundamentally important in the end, but something like that.

Reply:
ok as long everything follows the same convention :ok_hand: